### PR TITLE
Added social links in header 

### PR DESCRIPTION
**Pull request by gautmikhanna**

### CODE REVIEW

**Comment by pulkit-30** [comment]
🫤

Screenshot 2022-08-20 at 12 35 56 AM

File: dev.html [hunk 1]
```diff
@@ -32,10 +32,20 @@
...
+          ></a>
+          <a href="https://www.instagram.com/nexus_whatthehack/" target="_blank"
+            ><i class="fa-brands fa-instagram"></i
+          ></a>
+          <a
+            href="https://www.linkedin.com/company/nexus-what-the-hack/about/"
+            target="_blank"
+            ><i class="fa-brands fa-linkedin-in"></i
+          ></a>
+          <a href="https://discord.gg/RHn4CPuW" target="_blank"
```
Comment (pulkit-30) [inline dev.html:46]: LGTM 🎉 , only see discord invitation link @gautmikhanna

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by dewank07** [comment]
https://discord.gg/R53QVZJ2fn  use this link as we will be redirecting the audience to Elixir server already

**Review by Anv3sh — APPROVED** [review]
Changed the discord link looks good 👍🏻

File: dev.html (hunk 1 above)
Comment (Anv3sh) [reply dev.html]: Fixed it. 👍🏻

**Pull request merged**